src/Components/Board.js

```diff
@@ -1,43 +1,82 @@
-import Post from './Post'
-import Button from './Button'
-import { useState } from 'react'
+import { useState, useEffect } from "react";
+import { v4 as uuidv4 } from "uuid";
+import "./Board.scss";
+import Post from "./Post";
+import AddPostButton from "./AddPostButton";
+import axios from "axios";
 
 
-// install react-dnd and react-dnd-html5-backend
+const POST_WIDTH = 300;
+const POST_HEIGHT = 300;
 
+function Board() {
 
-// Wrap the container component with dnd Provider
+    const [ time, setTime] = useState([])
+    const [posts, setPosts] = useState([]);
 
+const onAddPost = () => {
+  const newPosts = [...posts];
 
-const Board = () => {
-    
+  axios
+    .get(
+      "https://rb7paj3rwfgcelny6aaqqagxna0idefd.lambda-url.us-east-1.on.aws/"
+    )
+    .then((response) => {
+      const timestamp = response.data;
+      const datetime = new Date(timestamp * 1000);
 
+      const post = {
+        id: uuidv4(),
+        x: Math.floor(Math.random() * (window.innerWidth - POST_WIDTH)),
+        y: Math.floor(Math.random() * (window.innerHeight - POST_HEIGHT)),
+        body: "",
+        datetime: datetime,
+      };
+      newPosts.push(post);
+      setPosts(newPosts);
+    });
+};
 
+    const onPostDrag = (index, state) => {
+        const newPosts = [...posts];
+        newPosts[index].x += Math.round(state.delta[0]);
+        newPosts[index].y += Math.round(state.delta[1]);
+        setPosts(newPosts);
+    };
```

Comment:
just wanted to take a sec and say that this drag code for the post-it notes is very cool